client/js/render.js

```diff
@@ -101,6 +101,20 @@ function buildChatMessage(msg) {
 		template = "msg_unhandled";
 	}
 
+	// Make the MOTDs a little nicer if possible
+	if (msg.type === "motd") {
+		let lines = msg.text.split("\n");
+
+		// If all non-empty lines of the MOTD start with a hyphen (which is common
+		// across MOTDs), remove all the leading hyphens.
+		if (lines.every((line) => line === "" || line === "-" || line.startsWith("- "))) {
```

Comment:
Re: `line === "-" || line.startsWith("- ")`, I went for explicit "all lines start with a dash+space or they are just one dash" to be conservative, but I guess I could do that, that should reasonably work.

Re: empty lines, I have to do this check (even more so with `line[0]`) because after the split, in some cases (such as Freenode) the last item is an empty string).

Reply:
Done.